Classes/Utility/CloudinaryApiUtility.php

```diff
@@ -31,7 +31,9 @@ public static function initializeByConfiguration(array $configuration)
                 'api_key' => $configurationService->get('apiKey'),
                 'api_secret' => $configurationService->get('apiSecret'),
                 'timeout' => $configurationService->get('timeout'),
-                'secure' => true
+                'secure' => true,
+                'private_cdn' => $configurationService->get('cname') !== '',
```

Comment:
$configurationService->get('cname') will throw an exception if `cname` is set to `env(…)` but the env does not exist. Is this intended?

Reply:
IMHO this is intended. As a reference to a non-existent environment variable is a clear configuration error.

The exception thrown

```php
throw new \RuntimeException(sprintf('No value found for environment variable "%s"', $matches[1]), 1626948978);
```

helps to find and fix this error.